kserve-module/tests/scripts/setup-cluster.sh

```
@@ -0,0 +1,386 @@
+deploy_kserve_module() {
+  log_info "Deploying kserve-module..."
+  create_or_skip_namespace "${KSERVE_NAMESPACE}"
+
+  local config_dir="${PROJECT_ROOT}/kserve-module/config"
+
+  local output
+  output=$(kustomize build "$config_dir" | sed "s|namespace: kserve|namespace: ${KSERVE_NAMESPACE}|g")
+
+  if [[ -n "${KSERVE_MODULE_IMG}" ]]; then
+    log_info "Using custom image: ${KSERVE_MODULE_IMG}"
+    output=$(echo "$output" | sed "s|image: .*kserve-module-controller.*|image: ${KSERVE_MODULE_IMG}|g")
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)